homeassistant/components/cover/__init__.py

```diff
@@ -168,19 +169,28 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     await component.async_setup(config)
 
     component.async_register_entity_service(
-        SERVICE_OPEN_COVER, {}, "async_open_cover", [CoverEntityFeature.OPEN]
+        SERVICE_OPEN_COVER,
+        {vol.Optional(ATTR_SPEED): vol.All(vol.Coerce(int), vol.Range(min=1, max=255))},
```

Comment:
This PR is changing our entity model. Before we can review this PR there needs to be approval in a discussion in our architecture repository.

https://github.com/home-assistant/architecture/discussions

https://developers.home-assistant.io/docs/core/entity#changing-the-entity-model

Reply:
If I remove this part from the PR, only implementing the per-device speed setting, any chance this PR is going to move forward independently of this architecture discussion?

Reply:
Setting a default speed is ok. Please open a new PR in that case that doesn't touch any core files so that the core team isn't notified.